packages/backend-modules/payments/graphql/resolvers/_mutations/cancelMagazineSubscription.ts

```diff
@@ -30,19 +33,24 @@ export = async function cancelMagazineSubscription(
     throw Error('api/unexpected')
   }
 
-  Auth.Roles.ensureUserIsMeOrInRoles(
-    await getSubscriptionOwner(ctx, sub),
-    ctx.user,
-    ['admin', 'supporter'],
-  )
+  const owner = await getSubscriptionOwner(ctx, sub)
+  if (!owner) {
+    throw Error('api/unexpected')
+  }
 
-  const cs = new CancellationService(new PaymentService(), ctx.pgdb)
+  Auth.Roles.ensureUserIsMeOrInRoles(owner, ctx.user, ['admin', 'supporter'])
+
+  const cs = new CancellationService(new PaymentService(), ctx.pgdb, [
+    new CancallationSlackNotifier(),
```

Comment:
also nitpicky: would it make sense here to instantiate the CancallationSlackNotifier inside the constructor of the CancellationService instead of passing it? Like that you wouldn't have to pass it as argument every time and I feel like it'd be more readable